flake8.rc

```diff
@@ -0,0 +1,19 @@
+[flake8]
+max_line_length = 80
+ignore =
```

Comment:
I suggest to enable all these checks. If not now, then in a second PR that fixes them.
None of them hurts, and all of them lead to more consistent code, if only optically.